tests/cargo-kani/proptest-lib/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! basic proptest examples. First one fails but second does not.
+
+#[cfg(test)]
+mod test {
+    use proptest::prelude::*;
+    proptest! {
+        /// Works fine. Shift has no overflow check.
+        fn works_fine(even_num in any::<u32>().prop_map(|x| x << 1)) {
+            prop_assert_eq!(even_num % 2, 0, "even number");
+        }
+    }
+
+    #[cfg(not(kani))]
+    mod expected {
+        proptest! {
+            /// Overflow by check in x * 2. TODO: Need to implement
+            /// expected fail tests. tracking issue #5
```

Comment:
Please paste the full URL to the issue.

Reply:
Fixed with d25e6d296c9bfb4c0e83171eeb5a6d75666d8b18